rtloader/common/log.h

```diff
@@ -0,0 +1,41 @@
+// Unless explicitly stated otherwise all files in this repository are licensed
+// under the Apache License Version 2.0.
+// This product includes software developed at Datadog (https://www.datadoghq.com/).
+// Copyright 2019 Datadog, Inc.
+#ifndef DATADOG_AGENT_RTLOADER_LOG_H
+#define DATADOG_AGENT_RTLOADER_LOG_H
+
+/*! \file log.h
+    \brief RtLoader log builtin header file.
+
+    The prototypes here provide functions to logs messages from rtloader to the
+    agent logger.
+*/
+
+#include <rtloader_types.h>
```

Comment:
Note: Non system header generally use quotes => `#include "rtloader_types.h"` https://stackoverflow.com/questions/21593/what-is-the-difference-between-include-filename-and-include-filename